torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -71,7 +71,7 @@ def resize_bounding_box(
 ) -> torch.Tensor:
     old_height, old_width = old_image_size
     new_height, new_width = new_image_size
-    return (
+    return (  # Shouldn't we be using a low-level kernel instead, similar to above?
```

Comment:
Not sure I understand. This is the low level kernel. I'm not aware of a resize kernel for bounding boxes that I could import from anywhere.

Reply:
There is an existing method:
https://github.com/pytorch/vision/blob/dad6e6a44d63272459b54175c92ed22955eb4207/torchvision/models/detection/transform.py#L289-L302

But I agree, probably we want to rewrite it here.